sdks/python/apache_beam/ml/inference/sklearn_inference.py

```diff
@@ -74,8 +77,11 @@ def load_model(self) -> BaseEstimator:
     return _load_model(self._model_uri, self._model_file_type)
 
   def run_inference(
-      self, batch: Sequence[numpy.ndarray], model: BaseEstimator,
-      **kwargs) -> Iterable[PredictionResult]:
+      self,
+      batch: Sequence[numpy.ndarray],
+      model: BaseEstimator,
+      extra_kwargs: Optional[Dict[str,
```

Comment:
No. Let's not put these extra args into sklearn.  The model doesn't want them, they don't need to be there.

This will be extra unused parameters that don't need to be there. They don't need to be here and they don't need to be in tensorflow implementations.

Don't put them here. If anything the only thing this PR should do is remove **kwargs from this interface, it shouldn't be here.

If someone puts extra args into this interface that it isn't expecting it should fail instead of work silently.

Reply:
> If someone puts extra args into this interface that it isn't expecting it should fail instead of work silently.

A ModelHandler that doesn't support extra_kwargs could raise an error. We don't need to rely on Python argument matching to raise the error.

Reply:
Good point. Removed.